docs/develop/build-and-run/cli.md

```diff
@@ -30,18 +30,18 @@ If users install the WasmEdge from the install script with the option `-e tf,ima
   * The `wasmedge` tool with TensorFlow-Lite, and `wasmedge-image` extensions.
   * Only on `x86_64` and `aarch64` Linux platforms, Android, and `x86_64` MacOS.
 
-The `wasmedge` CLI tool will execute the WebAssembly in ahead-of-time(AOT) mode and interpreter mode. It depends on if you use `wasmedgec` to AoT compile the wasm file.
+The `wasmedge` CLI tool will execute the WebAssembly in ahead-of-time(AOT) mode and interpreter mode if you use `wasmedgec` to AoT compile the wasm file.
```

Comment:
if we u use `wasmedgec` to AoT compile the wasm file, then WasmEdge is in the AoT mode.

Reply:
Okay, thanks for pointing that out!